mobile/src/screens/account.js

```diff
@@ -132,8 +132,13 @@ class AccountScreen extends Component {
             <Address
               address={address}
               label={fbt('Address', 'AccountItem.address')}
-              chars={10}
-              styles={{ fontSize: 16, marginVertical: 20 }}
+              chars={99999}
```

Comment:
Was this meant to be changed?

Reply:
It 's wrong. I restore these changes on this file.